docs/Git Guidelines.md

```diff
@@ -47,18 +46,18 @@ git checkout -b your-branch-name-here
 	- Carry out testing, if required
 - Once you are happy with the code, you can approve the PR from the Files Changed tab on GitHub, clicking Review Changes, and choosing Approve 
 - **Note: PRs require at least 1 approval before merging. This can be bypassed by admins but it is recommended to get 1 approval before merging**
-### Rebasing
-- When it comes time to merge your branch, rebase your branch with the latest main before doing so. 
-	- Note: squashing your commits may make this easier, but is not enforced. the decision to/not to squash is entirely up to you
+### Rebasing & Conflicts
+- When it comes time to merge your branch, you may rebase your branch with the latest main before doing so. 
+	- Note: squashing your commits may make this easier, but is not enforced. The decision to/not to squash is entirely up to you
 - Ensure you have the latest version of the main branch
 ```
 git checkout main
 git pull
 ```
-- Checkout your branch and rebase
+- Checkout your branch and rebase the commits you've made onto main
 ```
 git checkout your-branch-name-here
-git rebase main
+git rebase --onto main your-branch-name-here HEAD~5 //replace 5 with the amount of commits on your baranch
```

Comment:
The rebase command has been replaced with rebase onto. I think this is why the currentSelectedCard props drilling changes disappeared. Rebase onto will take your commits and rebase them after the changes on main, ensuring you have everything on main (provided you have the latest main)

Reply:
Good spot